plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java

```diff
@@ -191,6 +191,32 @@ public final void testBundleVendor() throws IOException {
     }
     assertEquals("Google Inc.", vendor);
   }
+  
+  @Test
+  public final void testBundleActivationPolicyLazy() throws IOException {
+    String policy = getManifestAttributes().getValue("Bundle-ActivationPolicy");
+    assertEquals("lazy", policy);
+  }
+  
+  @Test
+  public final void testManifestVersion() throws IOException {
+    Attributes manifest = getManifestAttributes();
+    assertEquals("1.0", manifest.getValue("Manifest-Version"));
+    assertEquals("2", manifest.getValue("Bundle-ManifestVersion"));
+  }
+
+  @Test
+  public final void testBundleClasspath() throws IOException {
+    Attributes manifest = getManifestAttributes();
+    assertEquals(".", manifest.getValue("Bundle-ClassPath"));
```

Comment:
This test will break on bundles that include jars, like googleapis.  The common error is specifying jar files and forgetting to include ".".

It would also be useful to verify that all elements here are specified in the `build.properties` > `bin.includes`.